components/StyledLink.js

```diff
@@ -0,0 +1,45 @@
+import styled, { css } from "styled-components";
+import Link from "next/link";
+
+export const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #E29B17;
+  
+  &:hover {
+    color: #D19015;
+  }
+  
+  ${({ variant }) =>
+      variant === "btn-primary" &&
```

Comment:
it's a link, but the variant appears to stand for "button primary", how about only "primary"?